<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateUserPage</class>
 <widget class="QWidget" name="UpdateUserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Actualizar Usuario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="placeholderText">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="placeholderText">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="roleComboBox">
     <item>
      <property name="text">
       <string>Usuario</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Administrador</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
